### PR TITLE
feat(registry): add zls to registry

### PR DESCRIPTION
**Pull request by hverlin**

### CODE REVIEW

File: docs/lang/zig.md [hunk 1]
```diff
@@ -15,3 +15,14 @@ mise use -g zig@latest  # install latest zig
...
 
 See available versions with `mise ls-remote zig`.
+
+## zig Language Server
+
+The `zig` language server ([zls](https://github.com/zigtools/zls)) needs to be installed separately.
+You can install it with `mise`:
+
+```sh
+mise use -g zls@0.13
```
Comment (jdx) [inline docs/lang/zig.md:25]: why'd you put the version in here? this is just going to go out of date

File: registry.toml [hunk 2]
```diff
@@ -1022,6 +1022,7 @@ zephyr.backends = ["asdf:nsaunders/asdf-zephyr"]
 zig.backends = ["core:zig"]
 zigmod.backends = ["ubi:nektro/zigmod", "asdf:mise-plugins/asdf-zigmod"]
 zigmod.test = ["zigmod version", "{{version}}"]
+zls.backends = ["aqua:zigtools/zls", "ubi:zigtools/zls"]
```
Comment (jdx) [inline registry.toml:1025]: if you can add a test (verified with `mise test-tool zls`) it would be good

File: docs/lang/zig.md (hunk 1 above)
Comment (hverlin) [reply docs/lang/zig.md:25]: Will remove. I added it as it needs to be on the same version as the `zig` version. Given that on the page, it shows `mise use -g zig@0.13` I thought I would do the same for `zls`

Comment (jdx) [reply docs/lang/zig.md:25]: oh I see, that makes sense, go ahead and change it back

File: docs/lang/zig.md [hunk 3]
```diff
@@ -15,3 +15,14 @@ mise use -g zig@latest  # install latest zig
...
 
 See available versions with `mise ls-remote zig`.
+
+## zig Language Server
+
+The `zig` language server ([zls](https://github.com/zigtools/zls)) needs to be installed separately.
+You can install it with `mise`:
+
+```sh
+mise use -g zls
```
Comment (jdx) [inline docs/lang/zig.md]:
```suggestion
mise use -g zls@0.13
```

**Pull request merged**